introduction.md

```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut a justo eget lectus ornare auctor. Aenean cursus ultrices lacinia. Integer euismod felis in imperdiet ultricies. Integer purus est, consequat lacinia elit a, rutrum interdum velit. Vivamus a odio volutpat, vehicula diam eu, ornare dolor. Etiam rhoncus justo ligula, non rhoncus lectus lobortis in. Nunc dapibus velit vitae purus luctus vestibulum. Maecenas dapibus tellus orci, quis eleifend augue pretium et. Vestibulum nec ultrices felis, vel placerat dolor. Vestibulum id augue id nisi vulputate eleifend in sit amet ipsum. Aliquam convallis malesuada lacus, et auctor tellus iaculis sit amet. Morbi posuere nunc nec malesuada ultrices. Quisque pretium mi ut dui euismod pellentesque. Curabitur nec nisl massa.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.